omod/src/main/web/dashboardwidgets/relationships/relationships.controller.js

```diff
@@ -51,9 +51,7 @@ export default class RelationshipsController  {
         }
 
         //fetchPrivileges
-        this.openmrsRest.get('session', {
-            v: 'custom:(privileges:(name))'
-        }).then((response) => {
+        this.openmrsRest.get('session').then((response) => {
             this.getPrivileges(response.user.privileges);
```

Comment:
Did this merge go wrong?

Reply:
It looks like master... guess it's an old diff?

Reply:
This is a recent change we made to fix this: https://issues.openmrs.org/browse/RA-1719

Reply:
Thanks, good to know. I see some privilege related differences in the program status widgets in the distro I'm working on (none of the edit/delete et c features are shown). Should we update to the latest RESTWS? @mks-d

Reply:
yes, if you are using the latest snapshot of core apps, you'll need the latest snapshot of REST for this to still work... I do plan to release a new version of REST WS with the fix in the coming days.